client/src/store/modules/traffic.js

```diff
@@ -0,0 +1,38 @@
+import axios from '@/plugins/axios';
+
+const namespaced = true;
+
+const state = () => ({
+    entries: []
+});
+
+const getters = {
+    total: state => state.entries.length,
+    perPage: state => page => state.entries.filter(entry => entry.page === page).length,
+    perSource: state => source => state.entries.filter(entry => entry.source === source).length
+};
+
+const mutations = {
+    SET_ENTRIES: (state, entries) => (state.entries = entries)
```

Comment:
in my opinion better will be regular function

Reply:
@daniel-pupczyk-rmtcfm-com To be honest. I think this function won't be needed. I had conversation with Rafał and it seems to be incorrect to store data in store in this case.